README.Rmd

```diff
@@ -0,0 +1,104 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# grtsdb
+
+<!-- badges: start -->
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![Lifecycle: maturing](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
+![GitHub](https://img.shields.io/github/license/inbo/grtsdb)
+[![R build status](https://github.com/inbo/grtsdb/workflows/R-CMD-check/badge.svg)](https://github.com/inbo/grtsdb/actions)
+[![Codecov test coverage](https://codecov.io/gh/inbo/grtsdb/branch/master/graph/badge.svg)](https://codecov.io/gh/inbo/grtsdb?branch=master)
+![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/inbo/grtsdb.svg)
+![GitHub repo size](https://img.shields.io/github/repo-size/inbo/grtsdb.svg)
+<!-- badges: end -->
+
+The goal of `grtsdb` is to create as statially balanced sample based on the 'Generalised Random Tesselation Stratified' strategy.
+We store the base schema in an SQLite database.
+Store this database to make the sampling reproducible.
+Sampling the same database with the same parameters yields a stable sample.
+
+## Installation
+
+You can install the development version from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("remotes")
+remotes::install_github("inbo/grtsdb")
+```
+
+## Example
+
+This is a basic example.
+
+Connect to a database.
+
+```{r}
+tmp_copy <- tempfile(pattern = "grts", fileext = ".sqlite")
+if (system.file("grts.sqlite", package = "grtsdb") != "") {
+  file.copy(system.file("grts.sqlite", package = "grtsdb"), tmp_copy)
+}
+library(grtsdb)
+db <- connect_db(tmp_copy)
+```
+
+To extract a sample, you'll need to specify the bounding box in projected
+coordinates and the size of the grid cells.
+
+```{r example}
+bbox <- rbind(
+  c(0, 32),
+  c(0, 32)
+)
+extract_sample(grtsdb = db, samplesize = 10, bbox = bbox, cellsize = 1)
+```
+Repeating the sample yields the same results.
+
+```{r}
+extract_sample(grtsdb = db, samplesize = 10, bbox = bbox, cellsize = 1)
+```
+
+```{r echo = FALSE}
+drop_legacy_sites(grtsdb = db, level = 5)
+```
+
+You can add legacy sites to the sampling scheme.
+
+```{r}
+legacy <- rbind(
+  c(4, 4),
+  c(17, 6)
+)
```

Comment:
What happens if you supply coordinates of legacy sites that deviate from coordinates in the grts-grid? So what happens if you replace for instance 4 by 4.3 in the example here? 
Can this be handled internally by the function (choose the closest grid point)? Or should it be handled by the user beforehand?

Reply:
The function maps the legacy sites to the grid cells. The sample function will return the grid cell centroid instead of the exact location of the legacy site.